spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala

```diff
@@ -2387,6 +2387,8 @@ object QueryPlanSerde extends Logging with ShimQueryPlanSerde with CometExprShim
       case _: ArrayIntersect => convert(CometArrayIntersect)
       case _: ArrayJoin => convert(CometArrayJoin)
       case _: ArraysOverlap => convert(CometArraysOverlap)
+      case _ if expr.prettyName == "array_except" =>
```

Comment:
Is `ArrayExcept` available only for newer version?

Reply:
This feature is available starting from Spark version 2.4.0.

Reply:
Any reason you had to use `prettyName` instead of 
```
case _: ArrayExcept =>
```

Reply:
Fixed.